src/quo2/components/markdown/text.cljs

```diff
@@ -23,7 +23,8 @@
          {:text-align (or align :auto)}
          (if (:color style)
            style
-           (assoc style :color (if (= (theme/get-theme) :dark) colors/white colors/neutral-100)))))
+           (assoc style :color (or secondary-color
+                                   (if (= (theme/get-theme) :dark) colors/white colors/neutral-100))))))
```

Comment:
Instead of using `(if (= (theme/get-theme) :dark) colors/white colors/neutral-100)` statement for theme, you can use `(colors/theme-colors light-color dark-color)`

Reply:
Done, Thanks Omar.